File: Solution/BeerUp/Repo/Modeles/ModelesBeerUp/AchatFacture.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Repo.Modeles.ModelesBeerUp
{
    public class AchatFacture
    {
        public int FacId { get; set; }
        public string BieNom { get; set; }
        public string EtaNom { get; set; }
        public long BieNbVuAch { get; set; }
        public long EtaNbVuAch { get; set; }
        public decimal BiePrix { get; set; }
        public decimal EtaPrix { get; set; }

        public Guid OrgId { get; set; }

        public Guid AchId { get; set; }

    }
}

File: Solution/BeerUpWeb/src/app/models/achat-facture.ts
export class AchatFacture {
    facId:number;
    bieNom:string | undefined;
    etaNom:string | undefined;
    bieNbVuAch:number | undefined;
    etaNbVuAch:number | undefined;
    biePrix:number | undefined;
    etaPrix:number | undefined;
    achId:string;
    orgId:string;

    constructor(){
        this.achId="";
        this.orgId="";
        this.facId=0;  
    }
}
